deploy/kubernetes/deploy-kiali-to-kubernetes.sh

```diff
@@ -236,18 +249,31 @@ do
 done
 
 # Deploy Kiali MonitoringDashboards to Kubernetes
-# Note for undeploy script: dashboards are implicitly undeployed when the related CRD is removed
+# Note: when undeploying, dashboards are implicitly undeployed when the related CRD is removed
 echo "Deploying Kiali dashboards to Kubernetes namespace ${NAMESPACE}"
-yaml_path="${YAML_DIR}/../dashboards/all.yaml"
-if [ -f "${yaml_path}" ]; then
-  echo "Using YAML file: ${yaml_path}"
-  cat ${yaml_path} | envsubst | kubectl apply -n ${NAMESPACE} -f -
+index_path="${YAML_DIR}/../dashboards/dashboards.txt"
+if [ -f "${index_path}" ]; then
+  echo "Using dashboards index file: ${index_path}"
+  dashboards=$(cat ${index_path})
 else
   get_downloader
-  yaml_url="https://raw.githubusercontent.com/kiali/kiali/${VERSION_LABEL}/deploy/dashboards/all.yaml"
-  echo "Downloading YAML via: ${downloader} ${yaml_url}"
-  ${downloader} ${yaml_url} | envsubst | kubectl apply -n ${NAMESPACE} -f -
+  index_url="https://raw.githubusercontent.com/kiali/kiali/${VERSION_LABEL}/deploy/dashboards/dashboards.txt"
+  echo "Downloading dashboards index file via: ${downloader} ${index_url}"
+  dashboards=$(${downloader} ${index_url})
 fi
-if [ "$?" != "0" ]; then
-  echo "NOTE: Could not deploy the runtimes dashboards. They are not mandatory and won't prevent Kiali to work. If you want to monitor your application runtimes, you can still deploy dashboards manually."
+
+if [ "${dashboards}" != "" ]; then
+  echo "=== Dashboards to install ==="
+  echo "${dashboards}"
+  echo "============================="
+  for dash in ${dashboards}
+  do
+    apply_dashboard ${dash}
+
+    if [ "$?" != "0" ]; then
+      echo "NOTE: Could not deploy runtimes dashboard [${dash}]. Dashboards are not mandatory and won't prevent Kiali to work. If you want to monitor your application runtimes, you can still deploy dashboards manually."
```

Comment:
Didn't you want to have just 1 occurrence of that message if several/all dashboards are missing? This is what I understood from previous reviews... but I'm fine either way :)

Reply:
I'm fine either way too -- but since it spits out one per file here, I was able to have the name of the dashboard that failed to load in the error message, so it is more clear what could not be found.